include/sancus_support/timer.h

```diff
@@ -73,5 +83,45 @@ void timerA_isr_entry(void)                                         \
             :::);                                                   \
 }
 
+#define TIMER_ISR_ENTRY2(fct)                                       \
```

Comment:
cf. comment above: a common top-level macro could be used here to avoid unnecessary code duplication?